packages/css/src/components/card/card.scss

```diff
@@ -79,6 +79,27 @@
       grid-column: 2 /-1;
     }
   }
+
+  &.dso-has-image {
```

Comment:
Ik zie hier best wel veel overlap... Eigenlijk heel veel overlap. Kunnen we hier niet wat fuseren?

Reply:
Ik heb ze nu samengevoegd dmv. extends